src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -694,133 +698,136 @@ New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
   </Collapser>
   <Collapser
     className="freq-link"
     id="http-errors"
     title="HTTP errors"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
   </Collapser>
   <Collapser
     className="freq-link"
     id="geography"
     title="Geography"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
   </Collapser>
   <Collapser
     className="freq-link"
     id="carriers"
     title="Carriers"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
   </Collapser>
   <Collapser
     className="freq-link"
     id="connect-types"
     title="Connection types"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
   </Collapser>
   <Collapser
     className="freq-link"
     id="versions"
     title="Versions"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀
   </Collapser>
   <Collapser
     className="freq-link"
     id="monthly-uniq"
     title="Monthly uniques"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀
   </Collapser>
   <Collapser
     className="freq-link"
     id="network"
     title="Network"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
   </Collapser>
   <Collapser
     className="freq-link"
     id="install"
     title="Installation"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
   </Collapser>
   <Collapser
     className="freq-link"
     id="upgrade"
     title="Upgrade"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
   </Collapser>
 </CollapserGroup>
 
 ## Related entities [#relat-ent]
 
 New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 🟡 Limited support
 
-This view shows:
+This view shows, for APM and/or OpenTelemetry services or hosts:
 
-* Relationships between services (APM and OpenTelemetry services)
-* Relationships between services and infrastructure (not supported for OpenTelemetry services)
+* Relationships between services
+* Relationships between services and infrastructure
```

Comment:
This and the next line are now up-to-date with respect to how entity relationships are created. OTel hosts work, as do uninstrumented external services. It's just service-to-database that's not done yet for OTel.

Reply:
@rkbodenner Looks good to me!